src/plugins/saved_objects_management/public/management_section/objects_table/components/import_mode_control.tsx

```diff
@@ -109,15 +104,11 @@ export const ImportModeControl = ({
       options={[overwriteEnabled, overwriteDisabled]}
       idSelected={overwrite ? overwriteEnabled.id : overwriteDisabled.id}
       onChange={(id: string) => onChange({ overwrite: id === overwriteEnabled.id })}
-      disabled={createNewCopies && !isLegacyFile}
+      disabled={createNewCopies}
       data-test-subj={'savedObjectsManagement-importModeControl-overwriteRadioGroup'}
     />
   );
 
-  if (isLegacyFile) {
-    return overwriteRadio;
-  }
```

Comment:
NIT: Now that this condition is removed, we no longer need to use this `overwriteRadio` variable, and we can directly inline it's value inside the returned value

Reply:
Done in [`317a448` (#110738)](https://github.com/elastic/kibana/pull/110738/commits/317a448ce18a0449c21b6ba620efaf2541835da6#diff-126e4960ff35b1e895e68c8d97b486e70379cf4e3da1766ab6942288299ce265)